backstop/tests/table.html

```diff
@@ -688,113 +687,109 @@ <h2 id="default">Default</h2>
             <div class="iui-tables-cell iui-tables-head iui-active-sort">
```

Comment:
Above checkbox svg is not available in icons, I guess? We use circle ones in
checkbox.html, location-marker.html, non-ideal-state.html, progress-indicator.html, radio.html

Reply:
The SVG used inside of the checkbox component is not included within the @itwin/itwinui-icons package because it is unusable without the CSS that chooses what path to show (`.iui-check-partial` VS `.iui-check`).

I'm not sure I understand what you mean by "we use circle ones".